tests/yast2_gui/yast2_bootloader.pm

```diff
@@ -38,15 +38,23 @@ sub run {
 
     #	bootloader options and set probe foreign OS, timeout
     assert_and_click 'yast2-bootloader_bootloader-options';
+
+    wait_still_screen 10;
+    send_key 'alt-t';    # Import untrusted GnuPG key -- workaround
+    wait_still_screen 10;
+
     assert_screen 'yast2-bootloader_bootloader-options-switched';
     send_key 'alt-b';
     wait_still_screen 3;
     send_key 'alt-t';
     type_string '16';
 
     #	default boot section
+    for (1 .. 2) { send_key 'alt-f10' }
```

Comment:
I don't think this loop is needed, I haven't seen in previous occurrences of the workaround, could you check please?

Reply:
here it is: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14342/files#diff-e5ae008bc24ec64aac7d7f80f7671324b746ffe329c8fd6c8e5dba662f942ae7

Reply:
it is needed to always send an even number of `'alt-f10'` because the needle will match the unmaximized version of the yast screen, but not the maximized

Reply:
I think we should also use send_key_until_needlematch('yast2-bootloader_default-boot-section_tw', 'alt-f10', 9, 2);
as otherwise we randomly end-up fullscreen (or is there some reason not to ?)

Reply:
> I think here we should also use `send_key_until_needlematch('yast2-bootloader_default-boot-section_tw', 'alt-f10', 9, 2);` as otherwise we randomly end-up fullscreen, see my other comment just below. Or is there some reason not to ?

I see that here https://openqa.nue.suse.com/tests/9386473#step/installation/59, we end-up full screen even if using send_key_until_needlematch, so I was wrong assuming it can be because of `for (1 .. 2)` , resolving conversation.

Reply:
code changed, but there is a failure. http://d218.qam.suse.de/tests/157#step/yast2_bootloader/28

Reply:
Sorry, I had closed this conversation with the following comment:
_I see that here https://openqa.nue.suse.com/tests/9386473#step/installation/59, we end-up full screen even if using send_key_until_needlematch, so I was wrong assuming it can be because of for (1 .. 2) , resolving conversation._
By that I meant you did not have to change that part of the code. , when we click "resolve conversation", it means no longer action is necessary.

Now if you would want to use send_key_until_needlematch here to be more consistent, you should use this:
`record_soft_failure('bsc#1191112 - Resizing window as workaround for YaST content not loading');
send_key_until_needlematch('yast2-bootloader_bootloader-options', 'alt-f10', 9, 2)`,
Sorry for not being enough precise.